src/Widgets/NavigationWidget/NavigationWidgetComponent.tsx

```diff
@@ -1,16 +1,29 @@
-import { provideComponent, Obj } from 'scrivito'
+import { provideComponent, Obj, currentPage } from 'scrivito'
 import Navbar from 'react-bootstrap/Navbar'
 
 import { NavigationWidget } from './NavigationWidgetClass'
 import { Brand } from './SubComponents/Brand'
 import { MainNavigation } from './SubComponents/MainNavigation'
 import { MetaNavigation } from './SubComponents/MetaNavigation'
 import { isHomepage } from '../../Objs/Homepage/HomepageObjClass'
+import { isLandingPage } from '../../Objs/LandingPage/LandingPageObjClass'
 
 provideComponent(NavigationWidget, ({ widget }) => {
   const root = Obj.root()
   if (!isHomepage(root)) return null
 
+  if (isLandingPage(currentPage())) {
```

Comment:
* [ ] FYI In the previous commit we made the LandingPage configurable regarding hide-from-nav and exclude-from-search. Here, we decide for a layout depending on the obj class. I find this a little inconsistent, since it's not clear why we want to have landing pages that appear in the nav, but don't allow landing pages to show the nav. We could think about
  1. making the nav appearance configurable, too. Or
  2. making the hide-from-search and hide-from-nav depend on the obj class